### PR TITLE
Change rails version from 6 to 6.1.3.1

### PR DESCRIPTION
**Pull request by earnestaddae**
New rails update. Use this branch rather. Thanks

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 ruby '2.7.1'
 
 gem 'puma'
-gem 'rails', '~> 6'
+gem 'rails', '~> 6.1.3.1'
```
Comment (TayKangSheng) [inline Gemfile:6]: Can you set `~> 6.1` instead so that we can allow for future patch versions.

Comment (earnestaddae) [reply Gemfile]: Just did for your perusal.

**Review by TayKangSheng — APPROVED** [review]
I think this looks good

**Comment by TayKangSheng** [comment]
Rails has already been updated in https://github.com/rubysg/rubysg-reboot/pull/328

**Pull request closed without merging**